src/_scss/pages/award/_moreButton.scss

```diff
@@ -0,0 +1,31 @@
+button.see-more {
+
+    @include button-unstyled;
+    color: $color-primary;
+
+    &:hover,
+    &:active {
+        color: $color-primary-darker;
+        text-decoration: underline;
+    }
+
+    &:visited {
+        color: $color-visited;
+    }
+
+    &:focus {
+        box-shadow: $focus-shadow;
+        outline: 0;
+    }
+    margin-top: .5rem;
```

Comment:
Removing `margin-top: .5rem;` will keep the `line-height` consistent on the line in which the More/Less link is displayed